src/findseed.c

```diff
@@ -0,0 +1,392 @@
+/**
+ * Copyright (C) 2024  Pancake
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+#include <stdlib.h>
+#include <signal.h>
+#include <string.h>
+
+#include <concord/discord.h>
+#include <concord/log.h>
+
+/// Config file for the bot
+#define CONFIG_FILE "config.json"
+
+/// Empty emoji
+#define NOTHING_EMOJI "<:nothing:912781172063490050>"
+/// Full end portal frame emoji
+#define FRAME_FULL_EMOJI "<:end_portal_frame_full:912780807091933184>"
+/// Empty end portal frame emoji
+#define FRAME_EMOJI "<:end_portal_frame:912780807041613884>"
+/// Lava emoji
+#define LAVA_EMOJI "<a:lava:912779051926700063>"
+/// Void emoji (top left)
+#define VOID_TL_EMOJI "<a:endportal1:968224841473855549>"
+/// Void emoji (top middle)
+#define VOID_TM_EMOJI "<a:endportal2:968224842224656424>"
+/// Void emoji (top right)
+#define VOID_TR_EMOJI "<a:endportal3:968224854639804457>"
+/// Void emoji (middle left)
+#define VOID_ML_EMOJI "<a:endportal4:968224854622998578>"
+/// Void emoji (middle middle)
+#define VOID_MM_EMOJI "<a:endportal5:968224854761414746>"
+/// Void emoji (middle right)
+#define VOID_MR_EMOJI "<a:endportal6:968224839246696448>"
+/// Void emoji (bottom left)
+#define VOID_BL_EMOJI "<a:endportal7:968224855826763926>"
+/// Void emoji (bottom middle)
+#define VOID_BM_EMOJI "<a:endportal8:968224856283951116>"
+/// Void emoji (bottom right)
+#define VOID_BR_EMOJI "<a:endportal9:968224854161645590>"
+/// Full broken end portal frame emoji
+#define FRAME_FULL_BROKEN_EMOJI "<:end_portal_frame_broken1:912811134191562812>"
+/// Broken end portal frame emoji
+#define FRAME_BROKEN_EMOJI "<:end_portal_frame_broken2:912811133952466974>"
+
+/// Corrupted texts
+#define CORRUPTED_TEXTS { \
+    "Great, you broke it.", \
+    "Wait? It's broken?!", \
+    "Okay.. let's pretend we never saw this", \
+    "Huh, that's weird. Let's hope it doesn't cause any troubles" \
```

Comment:
```suggestion
    "Okay... let's pretend we never saw this", \
    "Huh, that's weird. Let's hope it doesn't cause any trouble" \
```

Reply:
the 'that' doesn't need to be there because if the 'it' but I agree in the 's' at the end